mlir/utils/performance/tuningRunner.py

```diff
@@ -320,10 +320,10 @@ def main(args=None):
 
     # Note, appending results here to allow multiple config sets
     with open(parsed_args.output, 'a') as outFile:
-        print("# arch\ttestVector\tperfConfig", file=outFile)
+        print("# arch\tnum CUs\ttestVector\tperfConfig", file=outFile)
```

Comment:
super-nit : do we need the space between num and CU ?

Reply:
I guess it doesn't matter because this line is just a comment. But, yep, let me remove the white space for consistency

Reply:
We need the space to be consistent with the format of the following entries.

Reply:
I suppose we are adding this for the first time in these scripts (I maybe wrong) .. which entries are you talking about ?

Reply:
> super-nit : do we need the space between num and CU ?

I just removed the whitespace. As I mentioned before, line 323 generates just a comment which is getting ignored in migraphx. But, I still removed the whitespace for consistency